templates/idp/email-password-interaction/resetPassword.ejs

```diff
@@ -5,6 +5,17 @@
   <meta name="viewport" content="width=device-width, initial-scale=1"/>
   <title>Reset password</title>
   <link rel="stylesheet" href="/.well_known/css/styles/main.css" type="text/css">
+  <script>
+    const urlParams = new URLSearchParams(window.location.search);
+    const recordId = urlParams.get('rid');
+    window.onload = function() {
```

Comment:
That's antiquated 😉
```suggestion
    window.addEventListener('DOMContentLoaded', () =>
```

Reply:
This is what happens if you never write frontend javascript